examples/mobile/Cargo.toml

```diff
@@ -8,26 +8,10 @@ license = "MIT OR Apache-2.0"
 
 [lib]
 name = "bevy_mobile_example"
-crate-type = ["staticlib", "cdylib"]
```

Comment:
this is needed for iOS

Reply:
Will fix it right  now. And I'll change to `main` branch. I was actually curious about what I should base it on.